citesphere/src/main/webapp/WEB-INF/views/auth/authorities/list.html

```diff
@@ -49,25 +49,56 @@
 		$('#authority-delete-confirmation').modal("hide");
 		event.preventDefault();
 	});
+	
+	$("#displayAuthorities").change(function () {
+	    if($("#displayAuthorities").val() === 'all'){
+	    	window.location.href = [[@{/auth/authority/list}]];
+	    } else if($("#displayAuthorities").val() === 'userSpecific'){
+	    	window.location.href = [[@{/auth/authority/user/list}]];
+	    } else {
+	    	var zoteroGroupId = $("#displayAuthorities").val();
+	    	window.location.href = [[@{/auth/authority/}]] + zoteroGroupId + "/list";
+	    }
+	});
 });
 </script>
 </head>
 
 <body>
 <div layout:fragment="content">
 <h2>Managed Authority Entries</h2>
+<label for="displayAuthorities">Display Authorities</label>
+
+<select class="form-control" id="displayAuthorities" name="displayAuthorities" size="1">
+	<option th:selected="${#strings.equals(displayBy, 'all')}" value="all">All</option>
+	<option th:selected="${#strings.equals(displayBy, 'userSpecific')}" value="userSpecific">[[${username}]]</option>
+	<span th:each="groupDetail:${groups}">
+		<option th:selected="${#strings.equals(displayBy, groupDetail.groupId)}" th:value="${groupDetail.groupId}">Group - [[${groupDetail.name}]]</option>
+	</span>  
+</select><br><br> 
+	
 <table class="table table-striped table-bordered">
 <tr>
 	<th width="20%">URI</th>
 	<th>Name</th>
 	<th>Created on</th>
+	<th>Group</th>
 	<th></th>
 	<th></th>
 </tr>
 <tr th:id="'tr-'+${authority.id}" th:each="authority : ${authorities}">
 	<td><span th:text="${authority.uri}"/></td>
 	<td><span th:text="${authority.name}"/></td>
 	<td><span class="date" th:text="${authority.createdOn}"/></td>
+	<td>
+	<span th:if="${#lists.isEmpty(authority.groups)}"> - </span>
+	<span th:each="group,status : ${authority.groups}">
+		<span th:each="groupDetail:${groups}">
+			<span th:if="${#strings.equals(group, groupDetail.groupId)}">[[${groupDetail.name}]]</option>
+		</span>
+		<span th:if="${!status.last}">, </span>
+	</span>
+	</td>			
 	<td><a class="delete-link" href="" th:data-authority-id="${authority.id}" th:data-authority-name="${authority.name}" th:data-authority-uri="${authority.uri}"><i class="fas fa-trash-alt"></i></a></td>
 	<td><a th:href="@{${authority.id}+'/edit'}"><i class="far fa-edit" title="Edit"></i></a></td>
```

Comment:
This link doesn't work anymore now because it is relative and creates the wrong link when a group or the user has been selected.

Reply:
And neither does the delete link.